daemon/inspect.go

```diff
@@ -70,6 +70,7 @@ func (daemon *Daemon) ContainerInspectCurrent(ctx context.Context, name string,
 		if epConf.EndpointSettings != nil {
 			// We must make a copy of this pointer object otherwise it can race with other operations
 			apiNetworks[nwName] = epConf.EndpointSettings.Copy()
+			apiNetworks[nwName].DesiredMacAddress = ""
```

Comment:
Ah, so you're not really sending it through the API since you blank it out here and it has `omitempty`. I think I still prefer to make the json pkg ignore that field altogether -- and then, no need to make it empty here.

Reply:
Yes, indeed - but, as noted above, it does need to be JSON encoded so that the configured value is stored. So, to keep it out of the 'inspect' output, it has to be cleared here.